bridge/qb/client/functions.lua

```diff
@@ -153,17 +229,17 @@ function functions.SpawnVehicle(model, cb, coords, isnetworked, teleportInto)
     if cb then cb(veh) end
 end
 
----@deprecated use DeleteVehicle from imports/utils.lua
-functions.DeleteVehicle = DeleteVehicle
+---@deprecated use qbx.deleteVehicle from modules/lib.lua
+functions.DeleteVehicle = qbx.deleteVehicle
 
----@deprecated use GetPlate from imports/utils.lua
-functions.GetPlate = GetPlate
+---@deprecated use qbx.getVehiclePlate from modules/lib.lua
+functions.GetPlate = qbx.getVehiclePlate
```

Comment:
[`QBCore.Functions.GetPlate`](https://github.com/qbcore-framework/qb-core/blob/cdd1502f0157fe2c9e1d97ef1b6e79eceb9e6c9f/client/functions.lua#L386) handles the `vehicle` param differently in some cases. Maybe we should too to improve compatibility?

Reply:
Do we want it to just return end if there is no plate? I'd think it erroring out would be better than just doing that